sklearn/mixture/tests/test_dpgmm.py

```diff
@@ -128,6 +182,14 @@ class TestDPGMMWithFullCovars(unittest.TestCase, DPGMMTester):
     setUp = GMMTester._setUp
 
 
+@ignore_warnings(category=DeprecationWarning)
```

Comment:
you don't actually need these because ``assert_warns_message`` already catches the deprecation, IIRC.